src/exports.js

```diff
@@ -21,7 +21,6 @@ const StyleSheet = {
     create(sheetDefinition /* : SheetDefinition */) {
         return mapObj(sheetDefinition, ([key, val]) => {
             return [key, {
-                // TODO(gil): Further minify the -O_o--combined hashes
                 _name: process.env.NODE_ENV === 'production' ?
                     `_${hashObject(val)}` : `${key}_${hashObject(val)}`,
```

Comment:
Since this is going to be joined and hashed again, we can probably remove the leading underscore here now, right?